tests/factories.py

```diff
@@ -2,135 +2,167 @@
-
-        annotation = Annotation(tags=["foo", "bar"])
-
-    A random number is generated for each annotation and used in the URI
-    ("http://example.com/document_3"), document title ("Example Document 3"),
-    etc. To choose this number do:
-
-        annotation = Annotation(random_number=7)
-
-
-    """
-
-    class Meta:
-        model = storage.annotation_from_dict({}).__class__
-        exclude = ["username", "random_number", "num_tags", "exact_text",
-                   "document_title"]
-
-    username = "seanh"
-    random_number = factory.LazyAttribute(lambda n: random.randint(1, 10))
-    num_tags = None
-    exact_text = None
-    document_title = None
-
-    id = factory.Sequence(lambda n: "test_id_{n}".format(n=n + 1))
-    text = factory.Sequence(lambda n: "Test annotation {n}".format(n=n + 1))
-    user = factory.LazyAttribute(
-        lambda n: u"acct:{username}@127.0.0.1".format(username=n.username))
-
-    @factory.LazyAttribute
-    def created(stub):
-        # pylint: disable=no-self-use
-        return datetime.datetime.now().isoformat()
-
-    updated = factory.SelfAttribute('created')
-
-    @factory.LazyAttribute
-    def tags(stub):
-        num = (stub.num_tags if stub.num_tags is not None else
-               random.randint(1, 11))
-        return ["tag_{n}".format(n=n) for n in range(1, num + 1)]
-
-    @factory.LazyAttribute
-    def uri(stub):
-        return "http://example.com/document_{n}".format(n=stub.random_number)
-
-    @factory.LazyAttribute
-    def target_selectors(stub):
-        return [{"type": "RangeSelector",
-                 "startContainer": "/div[1]/article[1]/",
-                 "endContainer": "/div[1]/article[1]/section[1]",
-                 "startOffset": 0,
-                 "endOffset": 43},
-                {"start": 211,
-                 "end": 254,
-                 "type": "TextPositionSelector"},
-                {"type": "TextQuoteSelector",
-                 "prefix": "text quote prefix",
-                 "exact": (stub.exact_text if stub.exact_text is not None
-                           else "The exact text that was selected"),
-                 "suffix": "text quote suffix"},
-                {"type": "FragmentSelector",
-                 "value": ""}
-            ]
-
-    @factory.LazyAttribute
-    def permissions(stub):
-        return {
-            "admin": [stub.user],
-            "read": ["group:__world__"],
-            "update": [stub.user],
-            "delete": [stub.user]
-        }
-
-    @factory.LazyAttribute
-    def document(stub):
-        return {
-            "eprints": {},
-            "title": (stub.document_title or
-                      "Example Document {n}".format(n=stub.random_number)),
-            "twitter": {},
-            "dc": {},
-            "prism": {},
-            "highwire": {},
-            "facebook": {},
-            "link": [{"href": stub.uri}]
-        }
+from h.api import models as api_models
```

Comment:
Importing `models` for `update_document_metadata` is confusing to me. I think `from h.api.models.document import update_document_metadata` would be clearer.